code/modules/roguetown/roguecrafting/alchemy/reagents.dm

```diff
@@ -11,36 +11,34 @@
 	color = "#ff0000"
 	taste_description = "lifeblood"
 	overdose_threshold = 0
-	metabolization_rate = REAGENTS_METABOLISM * 3
+	metabolization_rate = REAGENTS_METABOLISM
 	alpha = 173
 
 /datum/reagent/medicine/healthpot/on_mob_life(mob/living/carbon/M)
 	if(volume > 0.99)
-		M.blood_volume = min(M.blood_volume+15, BLOOD_VOLUME_MAXIMUM)
-		M.adjustBruteLoss(-2*REM, 0)
-		M.adjustFireLoss(-2*REM, 0)
-		M.adjustOxyLoss(-1, 0)
+		M.blood_volume = min(M.blood_volume+20, BLOOD_VOLUME_MAXIMUM)
```

Comment:
so... you made this last 3x as long and heal 50% more?

Reply:
Yep, because previously prayer was just downright better. Now a drink of it is roughly equal to a base lesser miracle [3 oz/9 units is a swig from an alch flask.]

Reply:
them being worse then lesser miracle is fine considering you won't be able to get that cast on reliably in dungeons or out of town

Reply:
Healing from 27->15 per 9 u pushed.